### ISSUE TITLE
Cannot use plugin after codesign with AppSandboxing for publishing to Mac App Store

### ISSUE THREAD

**Issue by Cyhzault** [keijiro/FFmpegOut#51]
Hello,

First of all thank you for your FFmpeg integration plugin, it's been really helpful to handle in app video encoding and is quite easy to use. I successfully managed to integrate it in my application (on both OSX and Windows). I'm only struggling _after_ code signing my Application on OSX. 

I suspect the issue is linked to the App SandBoxing made mandatory by Apple on all Mac App store App. 

`com.apple.security.app-sandbox`

 The plugin works well when codesigning without Sandboxing. I tried multiple keys in entitlement including a solution using an app-sandbox inheritance but without any success.

`com.apple.security.inherit`

Details on my codesign process and my entitlement configuration can be found on a [stack overflow post i made](https://stackoverflow.com/questions/62041814/how-to-prevent-app-sandoxing-from-blocking-a-plugin-usage-after-codesigning).

Do you have any clues or hint on the matter ? Do i need a specific configuration on my entitlements file or my FFmpeg file structure ? 

Thanks again for your plugin and your time,
Regards,
Alexis.

**Comment by tomtomtong** [comment]
I also meet this issue. Have you solve this now?

**Comment by keijiro** [comment]
The problem is that I have zero knowledge about Mac App Store.

**Comment by tomtomtong** [comment]
Yes There is not much sharing about uploading app to Mac App Store in internet.
